tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract.txt

```diff
@@ -0,0 +1,20 @@
+REGEXP_EXTRACT([str0], '(E.{3})')
```

Comment:
we need to revert the deletion and renaming. 

line 1 of this file should go back in `tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.extract.txt`
lines 2 onwards can be in a new test file following the naming convention we talked about (adding `.full` or `.extended` eg tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.extract.full.txt`)